### ISSUE TITLE
ValueError: Cannot load file containing pickled data when allow_pickle=False

### ISSUE THREAD

**Issue by kzchen95** [justinjohn0306/FakeYou-Tacotron2-Notebook#16]
Getting this error when trying to train the model. Runtime error when first epoch starts.
Changed the 
`melspec = torch.from_numpy(np.load(filename))`
to
`melspec = torch.from_numpy(np.load(filename, allow_pickle=True))`
which still failed.

![image](https://github.com/justinjohn0306/FakeYou-Tacotron2-Notebook/assets/19570177/850c0fa1-8a87-49af-8207-b274f1a61d61)

**Comment by KyKhaw** [comment]
Any updates on this? I'm having the same issue

**Comment by KyKhaw** [comment]
Ok, this issue is related to the issue of transcripts not generating properly. I just changed the file paths in the list.txt file from .wav to .npy and it was resolved for me.

**Comment by jooshwa** [comment]
> Ok, this issue is related to the issue of transcripts not generating properly: #15
> 
> I just changed the file paths in the list.txt file from .wav to .npy and it was resolved for me.

Stuck here too. Is that all you did, just change the list.txt file? I tried that and got FileNotFoundError. I tried changing the actual .wav files as well (renamed them all .npy instead of .wav) but that gives me the same pickled data issue.

**Comment by KyKhaw** [comment]
Your .wav file should already have the .npy versions along with it. Did you run cell 6 to convert .wav files into mel spectrograms? Also be sure to change ALL filepaths in the list.txt file to .npy

**Comment by guidevops** [comment]
> Your .wav file should already have the .npy versions along with it. Did you run cell 6 to convert .wav files into mel spectrograms? Also be sure to change ALL filepaths in the list.txt file to .npy

Changing the list.txt to .npy in file paths fixed the problem for me.

**Comment by guidevops** [comment]
Right, I just discovered something: changing .npy to .wav causes the list.txt file to be saved empty. This happens because the function checks if the file referring to that transcription exists before saving the line in list.txt.
